src/components/FilledButton/DropdownHeaderButton.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import { Button, Popover, Typography } from '@material-ui/core'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import useStyles from './style'
+
+const blurAllOfClass = (className: string) => {
+  const els = document.getElementsByClassName(className)
+
+  for (const i in els) {
+    const el = els[i] as HTMLElement
+    if (!el.style) {
+      return
+    }
+    el.style.filter = 'blur(4px) brightness(0.4)'
+  }
+}
+
+const resetFiltersOfClass = (className: string) => {
+  const els = document.getElementsByClassName(className)
+
+  for (const i in els) {
+    const el = els[i] as HTMLElement
+    if (!el.style) {
+      return
+    }
+    el.style.filter = 'none'
+  }
+}
+
+export interface IProps {
+  name: string
+  classToBlur?: string
+  disabled?: boolean
+  onClick?: () => void
+  startIcon?: JSX.Element
+}
+export const DropdownHeaderButton: React.FC<IProps> = ({
+  name,
+  classToBlur = 'blur-at-overlay',
```

Comment:
I think we need global method aka blurContent and class blured should be stable.
Also i dont like that you iterate over all classes just make sure that static class i most important.